### ISSUE TITLE
setting env variable OCI_USER_OCID does not work

### ISSUE THREAD

**Issue by jesperberth** [oracle/oci-ansible-modules#97]
## Issue Report

**Describe the issue**

Setting export OCI_USER_OCID=ocid1.user.oc1..aaaxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx dosn't work, running playbooks fails with

```bash
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Invalid OCI configuration. Exception: {'user': 'missing'}"}
```

**Expected behavior**

Expecting playbook to run using env variables only

Setting ENV variables

```bash
export OCI_USER_FINGERPRINT=ax:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx
export OCI_USER_OCID=ocid1.user.oc1..aaaaaaaxxxxxxxxxxxxxxxxxxxxxxxxxxxxxlj6smukssdxds4q
export OCI_USER_KEY_FILE=/home/user/.oci/oci_api_key.pem
export OCI_TENANCY=ocid1.tenancy.oc1..aaaaaaaawxxxxxxxxxxxxxxxxxxxxxxxkcy3kgwun2pcbq7vj6k3xfbq
export OCI_REGION=eu-frankfurt-1
```

All but OCI_USER_OCID is acceptet

Playbook will run with .oci/config as below

```bash
[DEFAULT]
user=ocid1.user.oc1..aaaaaaaamfemxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx6smukssdxds4q
```

**Environment**

* OS version: Fedora 31 / Red Hat Enterprise Linux 7.7

* Ansible version: ansible 2.9.4

* OCI Python SDK version: 2.10.3

* OCI Ansible Modules version: 1.15.0

**Ansible playbook to reproduce the issue**

```bash
export OCI_USER_FINGERPRINT=ax:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx
export OCI_USER_OCID=ocid1.user.oc1..aaaaaaaxxxxxxxxxxxxxxxxxxxxxxxxxxxxxlj6smukssdxds4q
export OCI_USER_KEY_FILE=/home/user/.oci/oci_api_key.pem
export OCI_TENANCY=ocid1.tenancy.oc1..aaaaaaaawxxxxxxxxxxxxxxxxxxxxxxxkcy3kgwun2pcbq7vj6k3xfbq
export OCI_REGION=eu-frankfurt-1
```

```bash
(ansible) [user@ansible2 oci]$ cat get-route.yml
```

```bash
---
- name: OCI Webserver Network
  hosts: localhost
  vars:
    compartmentid: "ocid1.tenancy.oc1..aaaaaxxxxxxxxxxxxxxxxxxxxxxx3xfbq"

  tasks:
  - name: List Route Table
    oci_route_table_facts:
      compartment_id: "{{ compartmentid }}"
      vcn_id: 'ocid1.vcn.oc1.eu-frankfurt-1.amaaaxxxxxxxxxxxxxxxxxxxxx7a5hfimd5ta36ezusq'
```

**Running playbook**

```bash
(ansible) [user@ansible2 oci]$ cat ../.oci/config
[DEFAULT]
```

```bash
(ansible) [user@ansible2 oci]$ ansible-playbook get-route.yml -vv
ansible-playbook 2.9.4
  config file = None
  configured module search path = ['/home/user/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/user/ansible/lib/python3.7/site-packages/ansible
  executable location = /home/user/ansible/bin/ansible-playbook
  python version = 3.7.6 (default, Jan 30 2020, 09:44:41) [GCC 9.2.1 20190827 (Red Hat 9.2.1-1)]
No config file found; using defaults
[WARNING]: No inventory was parsed, only implicit localhost is available

[WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'


PLAYBOOK: get-route.yml *******************************************************************************************************************************************************************************************
1 plays in get-route.yml

PLAY [OCI Webserver Network] **************************************************************************************************************************************************************************************

TASK [Gathering Facts] ********************************************************************************************************************************************************************************************
task path: /home/user/oci/get-route.yml:2
ok: [localhost]
META: ran handlers

TASK [List Route Table] *******************************************************************************************************************************************************************************************
task path: /home/user/oci/get-route.yml:8
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Invalid OCI configuration. Exception: {'user': 'missing'}"}

PLAY RECAP ********************************************************************************************************************************************************************************************************
localhost                  : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0
```

**Running Playbook with user sat in .oci/config**

```bash
(ansible) [user@ansible2 oci]$ cat ../.oci/config
[DEFAULT]
user=ocid1.user.oc1..aaaaaaaamfemxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx6smukssdxds4q
```

```bash
(ansible) [user@ansible2 oci]$ ansible-playbook get-route.yml -vv
ansible-playbook 2.9.4
  config file = None
  configured module search path = ['/home/user/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/user/ansible/lib/python3.7/site-packages/ansible
  executable location = /home/user/ansible/bin/ansible-playbook
  python version = 3.7.6 (default, Jan 30 2020, 09:44:41) [GCC 9.2.1 20190827 (Red Hat 9.2.1-1)]
No config file found; using defaults
[WARNING]: No inventory was parsed, only implicit localhost is available

[WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'


PLAYBOOK: get-route.yml *******************************************************************************************************************************************************************************************
1 plays in get-route.yml

PLAY [OCI Webserver Network] **************************************************************************************************************************************************************************************

TASK [Gathering Facts] ********************************************************************************************************************************************************************************************
task path: /home/user/oci/get-route.yml:2
ok: [localhost]
META: ran handlers

TASK [List Route Table] *******************************************************************************************************************************************************************************************
task path: /home/user/oci/get-route.yml:8
ok: [localhost] => {"changed": false, "route_tables": [{"compartment_id": "ocid1.tenancy.oc1..aaaaaaaxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxbq7vj6k3xfbq", "defined_tags": {"Oracle-Tags": {"CreatedBy": "email@domain.com", "CreatedOn": "2020-02-11T19:12:26.581Z"}}, "display_name": "Default Route Table for WebsiteNetwork", "freeform_tags": {}, "id": "ocid1.routetable.oc1.eu-frankfurt-1.aaaaaaaaxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxt3lgzf5kulja", "lifecycle_state": "AVAILABLE", "route_rules": [], "time_created": "2020-02-11T19:12:26.584000+00:00", "vcn_id": "ocid1.vcn.oc1.eu-frankfurt-1.amaaaaaaxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx3ly7a5hfimd5ta36ezusq"}]}
META: ran handlers
META: ran handlers

PLAY RECAP ********************************************************************************************************************************************************************************************************
localhost                  : ok=2    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
```

**Comment by manojmeda** [comment]
@jesperberth Can you try setting `OCI_USER_ID` instead of `OCI_USER_OCID`. The documentation is wrong and we will update in the next release.

**Comment by jesperberth** [comment]
@manojmeda OCI_USER_ID did it,  playbooks runs without issues now.

Thanks

**Comment by gauravjain000** [comment]
Hey Jesper, I am the product manager on this product and would love to connect with you to understand how you use it and how we can improve your experience. If it is worth the time, shoot me an email at g.gaurav.j.jain@oracle.com

**Comment by AmeyaLokre** [comment]
This has been fixed in release v1.16.0
